src/data/faq.json

```diff
@@ -467,20 +467,26 @@
                                 "anchor": "eu_dcc_export",
                                 "active": false,
                                 "textblock": [
+                                    "<b>Updated ??? ?? ????</b><hr>",
                                     "<b>Purpose</b>",
                                     "As of version 2.10 of the Corona-Warn-App, you can export scanned EU Digital COVID Certificates into PDF documents. This function is primarily used to create backup copies of the certificates for your personal purposes.",
+                                    "As of version 2.24 of the Corona-Warn-App, you can save all vaccination and recovery certificates available in your app, as well as all tests that are not older than 72 hours, in a single PDF document.",
                                     "Of course, the original EU templates are used for the layout of the certificates.",
                                     "<b>Confidentiality</b>",
                                     "Exporting and saving the certificates is voluntary. The creation of the PDF file takes place exclusively locally on your smartphone. Only you have access to this document.",
                                     "However, please note that the generated document contains sensitive personal information. Therefore, please handle the PDF file with care. Show the document only to people you trust and who are authorized to review it.",
                                     "We strongly advise against sharing the PDF document via potentially insecure channels (e.g. by email or via other apps).",
                                     "<b>Limitation</b>",
                                     "With the new function, only PDF documents of certificates that were also issued in Germany can be generated. Otherwise, a detailed message appears: 'Display print version not possible - The print version of the certificate cannot be displayed because the certificate was not issued in Germany.'",

```

Comment:
"With the new function" is ambiguous because the article now covers two updates. Which one is the new one?

Change to

"When exporting a single certificate ..."